uint8 motorLeft
uint8 motorRight
uint8 weapon
